Assets/Scripts/Nodes/FindGod.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using System.Collections;
+
+using BehaviorTree;
+
+public class FindGod : LeafNode
+{
+	private Hashtable _data;
+
+	public override void InitSelf( Hashtable data )
+	{
+		_data = data;
+	}
+
+	public override NodeStatus TickSelf()
+	{
+		GodTag targetGod = GameObject.FindObjectOfType<GodTag>();
```

Comment:
Does this happen every frame? If so can we move this into InitSelf and have a callback that updates this when needed? Really skeptical of FindObject each frame.